CHANGELOG.md

```diff
@@ -1,12 +1,27 @@
 # CHANGELOG
 
-## v0.12.0-rc.0
+## v0.12.0
+*February 24th, 2022*
 
-> ⚠️  This is a release candidate and not the final release for v0.12.0.
+This release notably brings compatibility with Cosmos SDK 0.45 and IBC v3.0.0-rc.0,
```

Comment:
Should we be more specific about the SDK version i.e. `0.45.1`?

Reply:
I don't think it's required, we should be compatible with SDK 0.45.x.